docs/usage/types/types_fields.md

```diff
@@ -62,12 +62,13 @@ class UserIdentity(TypedDict, total=False):
 
 
 class User(TypedDict):
+    __pydantic_config__ = ConfigDict(extra='forbid')
+
     identity: UserIdentity
     age: int
 
 
 class Model(BaseModel):
```

Comment:
this change would mean slightly different behaviour, are you sure this is right?

Reply:
we can keep this line here. but it does not relate to the `ValidationError` below(second validation error regarding extra `email` key in input values).

The goal is to show extra `email` key is not permitted. but the doc test is passing because it does not fall into the `except` block. but with  this change and adding `__pydantic_config__` to `User` class, it will work